data/infra/resources/service.yaml

```diff
@@ -36,44 +36,48 @@ syntax_full_properties_list:
   `stop_command`, `supports`, `timeout`, and `user` are the properties available to
   this resource."
 actions_list:
-  :disable:
-    markdown: Disable a service. This action is equivalent to a `Disabled` startup
-      type on the Microsoft Windows platform. This action is not supported when using
-      System Resource Controller (SRC) on the AIX platform because System Resource
-      Controller (SRC) does not have a standard mechanism for enabling and disabling
-      services on system boot.
+  :nothing:
+    shortcode: resources_common_actions_nothing.md
   :enable:
     markdown: Enable a service at boot. This action is equivalent to an `Automatic`
       startup type on the Microsoft Windows platform. This action is not supported
       when using System Resource Controller (SRC) on the AIX platform because System
       Resource Controller (SRC) does not have a standard mechanism for enabling and
       disabling services on system boot.
-  :nothing:
-    shortcode: resources_common_actions_nothing_default.md
-  :reload:
-    markdown: Reload the configuration for this service.
-  :restart:
-    markdown: Restart a service.
+  :disable:
+    markdown: Disable a service. This action is equivalent to a `Disabled` startup
+      type on the Microsoft Windows platform. This action is not supported when using
+      System Resource Controller (SRC) on the AIX platform because System Resource
+      Controller (SRC) does not have a standard mechanism for enabling and disabling
+      services on system boot.
   :start:
     markdown: Start a service, and keep it running until stopped or disabled.
   :stop:
     markdown: Stop a service.
+  :restart:
+    markdown: 
+  :reload:
+    markdown: 
+  :mask:
+    markdown: 
+  :unmask:
+    markdown: 
```

Comment:
Looks like there's a discrepancy with the syntax example above.

Reply:
Weird. Now redacted